site/content/docs/concepts/pipelines.en.md

```diff
@@ -101,9 +103,9 @@ stages:
 ```
 You can see every available configuration option for `manifest.yml` on the [pipeline manifest](../manifest/pipeline.en.md) page.
 
-There are 3 main parts of this file: the `name` field, which is the name of your pipeline, the `source` section, which details the repository and branch to track, and the `stages` section, which lists the environments you want this pipeline to deploy to. You can update this anytime, but you must commit and push the changed files and run `copilot pipeline deploy` afterward.
+There are 3 main parts of this file: the `name` field, which is the name of your pipeline, the `source` section, which details the repository and branch to track, and the `stages` section, which lists the environments you want this pipeline to deploy or deploy to. You can update this anytime, but you must commit and push the changed files and run `copilot pipeline deploy` afterward.
 
-Typically, you'll update this file to add environments to deploy to, specify the order of deployments, add actions for the pipeline to run before or after deployment, or change the branch to track. You also may add a manual approval step before deployment or commands to run tests (see [Customization](#customization)). If you are using CodeStar Connections to connect to your repository and would like to utilize an existing connection rather than let Copilot generate one for you, you may add the connection name here.
+Typically, you'll update this file to add environments, specify the order of deployments, add actions for the pipeline to run before or after deployment, or change the branch to track. You also may add a manual approval step before deployment or commands to run tests (see [Customization](#customization)). If you are using CodeStar Connections to connect to your repository and would like to utilize an existing connection rather than let Copilot generate one for you, you may add the connection name here.
```

Comment:
What do you think of something like this:
```
Typically, you'll update this file to add stages to deploy each environment, or to deploy workloads into each environment, ...
```

Reply:
I was just trying to avoid "environments to deploy or deploy to" for a third time. I'm worried that "deploy each environment" and "into each environment" implies all of the environments...? 
Are you okay with me just changing this one to "add environments to deploy or deploy to" also?

Reply:
you know, it's not necessarily "add," either...."change"?

Reply:
updated